sympy/functions/elementary/exponential.py

```diff
@@ -596,7 +596,8 @@ def _eval_expand_log(self, deep=True, **hints):
 
     def _eval_simplify(self, ratio, measure):
         from sympy.simplify.simplify import expand_log, simplify
-        expr = self.func(simplify(self.args[0], ratio=ratio, measure=measure))
+        expr = self.func(simplify(self.args[0], ratio=ratio, measure=measure),
```

Comment:
`base` is known to eval but I think is ignored at instantiation; I'm not sure why you don't get an error (or maybe you will when tests run).  How about

```
expr = self.func(*[simplify(i, ratio=ratio, measure=measure)
    for i in self.args])
```